[feature][PUSH-18219]

nikolleto opened the pull request:
add possibility to set default notification channel name in Android;
add possibility to set notification channel description both via code and control panel in Android;

Commit:
[feature][PUSH-18219]
add possibility to set default notification channel name in Android;
add possibility to set notification channel description both via code and control panel in Android;

Commit:
[feature][PUSH-18219]
add UnityNotificationFactory with ability to connect callback to both change and localize channel name and channel description

AAEvstefeev commented:
Тестов бы по больше

Commit:
fix listeners;
fix factory;
update aars

Commit:
fix javadoc in OnAddNotificationChannelListener class

kaizerzer commented on PluginSource/Android/pushwoosh-unityplugin/build.gradle:
```diff
@@ -24,7 +24,7 @@ dependencies {
 	compile "com.android.support:support-v4:${support}"
 
-    compile "com.pushwoosh:pushwoosh:${pushwoosh}"
+    compile(name:'pushwoosh-5.13.2', ext:'aar')
```
нужно брать пушвуш из мавена

Commit:
use maven's 'pushwoosh' dependency instead of aar;
update pushwoosh version in Android's plugin source main build.gradle file;
update pushwoosh version in PushwooshDeoendencies.xml;
fix method naming violation in OnAddNotificationChannelListener.cs file

Commit:
remove flatDir{} method call from Android plugin source main build.gradle file

kaizerzer commented on PushwooshUnitySample/Assets/Pushwoosh/Editor/PushwooshDependencies.xml:
```diff
@@ -2,10 +2,10 @@
+        <androidPackage spec="com.pushwoosh:pushwoosh:5.14.0"/>      
+        <androidPackage spec="com.pushwoosh:pushwoosh-amazon:5.14.0"/>
+        <androidPackage spec="com.pushwoosh:pushwoosh-badge:5.14.0"/>
+        <androidPackage spec="com.pushwoosh:pushwoosh-location:5.14.0"/>
```
посмотри на 2 строчки вниз

Commit:
revert PushwooshDependencies.xml file

Commit:
refactor PushNotificator;
rename OnAddNotificationChannelListener to NotificationChannelDelegate;
